### PR TITLE
Added dependency on scalecube-benchmarks and refactoring

### PR DESCRIPTION
**Pull request by segabriel**

### CODE REVIEW

File: benchmarks/pom.xml [hunk 1]
```diff
@@ -24,6 +24,10 @@
       <artifactId>scalecube-services</artifactId>
       <version>${project.version}</version>
     </dependency>
+      <dependency>
+          <groupId>io.scalecube</groupId>
+          <artifactId>benchmarks-api</artifactId>
+      </dependency>
```
Comment (artem-v) [inline benchmarks/pom.xml]: Format xml file

File: pom.xml [hunk 2]
```diff
@@ -58,6 +58,7 @@
   <properties>
     <jackson.version>2.9.0</jackson.version>
     <scalecube.version>1.0.9</scalecube.version>
+      <scalecube-benchmarks.version>1.0.0</scalecube-benchmarks.version>
```
Comment (artem-v) [inline pom.xml]: Format xml

File: pom.xml [hunk 3]
```diff
@@ -212,6 +213,11 @@
       </dependency>
 
       <!-- Other libs -->
```
Comment (artem-v) [inline pom.xml:215]: Format xml

**Review by artem-v — CHANGES_REQUESTED** [review]
Cosmetic changes are needed

File: pom.xml (hunk 3 above)
Comment (segabriel) [reply pom.xml:215]: done

**Review by aharonha — APPROVED** [review]
:+1:

**Pull request merged**